http/headers/accept-ch.json

```diff
@@ -4,6 +4,7 @@
       "Accept-CH": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Accept-CH",
+          "spec_url": "https://datatracker.ietf.org/doc/html/rfc8942",
```

Comment:
To pass CI, you need a URL fragment. Something like https://datatracker.ietf.org/doc/html/rfc8942#section-3.1 might do, but I didn't actually read this closely so you might have a better link in mind.

Reply:
Thanks @ddbeck . You learn something new every day. Yes, that fragment is good, and I have inserted it.